### ISSUE TITLE
gtt ins't responding

### ISSUE THREAD

**Issue by AlejoFab** [kriskbx/gitlab-time-tracker#87]
Can't interact with the program, every command I write into cmd leads to nothing. The program doesn't respond and I must interrupt. I have installed the latest version, but have the same problem if I download and install a lower version, for example 1.7.39 or 1.7.37. Any ideas what the problem could be? Thanks for any respond

**Comment by AlejoFab** [comment]
Solved

**Comment by kriskbx** [comment]
Great you could solve it! What was the problem?

**Comment by AlejoFab** [comment]
I ran the commands in cmd on harddisk U: until I found out it has to be C:.
I thought it would work as well on U: as I installed gtt globally but I
could only see the list of options and call the version, but not the actual
commands like start, stop, log, etc.

I'm not sure why but as I have node.js installed on harddisk C: that could
make the difference. Maybe you know it and could explain it to me:)

Am Di., 19. Feb. 2019, 18:35 hat kris <notifications@github.com>
geschrieben:
